helpers.js

```diff
@@ -202,6 +202,9 @@ exports.api = (dispatch, url, params = {}) => {
   if (params.user && params.user.jwt) {
     params.headers.Authorization = `Bearer ${params.user.jwt}`
   }
+  if (params.getAllPages) {
```

Comment:
We shouldn't be fetching every endorsement... there's thousands of them. Let's just paginate the table 25 at a time.

I optimized the query, but fetching a every endorsement could potentially be megabytes of HTML/JSON.

Reply:
I'm not sure pages of only 25 results each is nearly as useful. Maybe 500?

Part of the appeal is specifically in being able to scroll through each and every entry, seeing just how long it is (and growing).

Reply:
Also the filtering logic is all happening on the frontend right now.
So need to move that to backend if only grabbing a partial selection of all the entries.